### DISCUSSION TITLE
ss58 申请一問

### DISCUSSION THREAD

**Discussion by jimmychu0807** [ParityAsia/substrate-newsletter#4] [Substrate/Polkadot 技术问题]
- 刚刚重新看substrate源码，ss58 prefix是用作什么的啊，跨链地址前缀的标记？
- 是不是只要pr上去可以申请了么？几个reserved是可以申请么，还是48之后的？
 
<img width="647" alt="WechatIMG167" src="https://user-images.githubusercontent.com/898091/107007772-f584a680-67cd-11eb-8172-7456b4e2d991.png">

\- 昊暠 skyh

**Comment by skyh24** [comment]
谢谢，有更新告知

**Comment by jimmychu0807** [comment]
@skyh24 

- https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)
- https://wiki.polkadot.network/docs/en/learn-accounts#address-format
- https://github.com/paritytech/substrate/blob/master/ss58-registry.json

这三个文件一起看吧。确实是每个网络有个独特的 ss58 prefix。至于怎么申请，理论上是作个 PR, Substrate 核心开发人员把它 merge 了就完事。问题是，要 merge 到 substrate 主代码里，可能你需要到 element 里和他们交流，或你开了 PR 后 他们会和你/你团队讨论。
